pages/usecases/vs-code-home-page/index.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
```

Comment:
This file should be in the root of `pages` directory if we're to add this page to B.io